src/components/Form2.jsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+// import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+
+const Form2 = () => {
+  const navigate = useNavigate();
+  // const dispatch = useDispatch();
+  // const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [passwordConfirmation, setPasswordConfirmation] = useState('');
+
+  // const handleChange = (e) => {
+  //   setName(e.target.value);
+  // };
+
+  const resetForm = () => {
+    setEmail('');
+    setPassword('');
+    setPasswordConfirmation('');
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    fetch(
+      'http://localhost:3001/registrations',
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          user: {
+            email,
+            password,
+            password_confirmation: passwordConfirmation,
```

Comment:
I think the fields required in the signup page are `name`, `username`, and `password`. There is no need for `email` and `password_confirmation` field